easybuild/easyconfigs/l/libwebp/libwebp-1.1.0-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'libwebp'
+version = '1.1.0'
+
+homepage = 'https://developers.google.com/speed/webp/'
+description = """WebP is a modern image format that provides superior
+lossless and lossy compression for images on the web. Using WebP,
+webmasters and web developers can create smaller, richer images that
+make the web faster."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = ['https://storage.googleapis.com/downloads.webmproject.org/releases/webp']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['98a052268cc4d5ece27f76572a7f50293f439c17a98e67c4ea0c7ed6f50ef043']
+
+builddependencies = [('binutils', '2.34')]
+
+dependencies = [
+    ('libjpeg-turbo', '2.0.4'),
+    ('libpng', '1.6.37'),
+    ('LibTIFF', '4.1.0'),
+    ('giflib', '5.2.1'),
+]
+
+configopts = "--enable-libwebpmux"
```

Comment:
@lexming do you need to include spaces here?

Reply:
not really, EB already puts spaces around `configopts`. At the easyconfig level, it is only necessary to put spaces whenever `configopts` is expanded.

Reply:
Yes, I went too far the other way after rightly being caught doing this badly recently...